daemons/based/based_common.c

```diff
@@ -216,7 +216,7 @@ static const cib_operation_t cib_server_ops[] = {
     },
     {
         PCMK__CIB_REQUEST_PRIMARY,
-        cib_op_attr_modifies|cib_op_attr_privileged|cib_op_attr_local,
+        cib_op_attr_privileged|cib_op_attr_local,
```

Comment:
Did you check all the usages of _modifies to see if some side effect is needed by REQUEST_PRIMARY?

Reply:
I thought I did but double-checking, there may be some legacy mode issues:
```
parse_local_options_v1():
    if (pcmk_is_set(operation->flags, cib_op_attr_modifies)
        && !pcmk_is_set(call_options, cib_inhibit_bcast)) {
        /* we need to send an update anyway */
        *needs_reply = TRUE;
    } else {
        *needs_reply = FALSE;
    }

parse_peer_options_v2():
    } else if (is_reply
               && pcmk_is_set(operation->flags, cib_op_attr_modifies)) {
        crm_trace("Ignoring legacy %s reply sent from %s to local clients", op, originator);
        return FALSE;
```
I'm not really sure that we need(ed) a reply for `REQUEST_PRIMARY`, but it might come in from a legacy node... that stuff is confusing. It's probably better to add a COMPAT comment to drop `cib_op_attr_modifies` from this request in the future.

Reply:
updated